inspire_dojson/journals/rules.py

```diff
@@ -27,88 +27,185 @@
+def license(self, key, value):
     return {
-        'title': value.get('a'),
-        'subtitle': value.get('b'),
+        'license': value.get('a'),
+        'url': value.get('u'),
+    }
+
+
+@journals.over('_harvesting_info', '^583..')
+def _harvesting_info(self, key, value):
+    return {
+        'coverage': value.get('a'),
+        'date_last_harvest': value.get('c'),
+        'last_seen_item': value.get('3'),
+        'method': value.get('i'),
     }
 
 
+@journals.over('public_notes', '^640..')
+@utils.for_each_value
+def public_notes_640(self, key, value):
+    public_note = value.get('a')
+
+    if public_note:
+        return {
+            'source': value.get('9'),
+            'value': public_note,
+        }
+
+
 @journals.over('publisher', '^643..')
 @utils.for_each_value
 def publisher(self, key, value):
     return value.get('b')
 
 
-@journals.over('short_titles', '^711..')
+@journals.over('_private_notes', '^667..')
 @utils.for_each_value
-def short_titles(self, key, value):
-    return {'title': value.get('a')}
+def _private_notes(self, key, value):
+    _private_note = value.get('x')
 
+    if _private_note:
+        return {
+            'source': value.get('9'),
+            'value': _private_note,
+        }
 
-@journals.over('title_variants', '^730..')
+
+@journals.over('doi_prefixes', '^677..')
 @utils.for_each_value
-def title_variants(self, key, value):
-    return {'title': value.get('a')}
+def doi_prefixes(self, key, value):
+    return value.get('d')
 
 
-@journals.over('related_records', '^78002')
+@journals.over('public_notes', '^680..')
 @utils.for_each_value
-def related_records(self, key, value):
-    record = get_record_ref(maybe_int(value.get('w')), 'journals')
+def public_notes_680(self, key, value):
+    public_note = value.get('i')
 
-    if record:
+    if public_note:
         return {
-            'curated_relation': record is not None,
-            'record': record,
-            'relation': 'predecessor',
+            'source': value.get('9'),
+            'value': public_note,
         }
+
+
+@journals.over('proceedings', '^690..')
+def proceedings(self, key, value):
+    proceedings = self.get('proceedings')
+    refereed = self.get('refereed')
+
+    if not proceedings:
+        normalized_a_values = [el.upper() for el in force_list(value.get('a'))]
+        if 'PROCEEDINGS' in normalized_a_values:
+            proceedings = True
+
+    if not refereed:
+        normalized_a_values = [el.upper() for el in force_list(value.get('a'))]
+        if 'PEER REVIEW' in normalized_a_values:
+            refereed = True
+        elif 'NON-PUBLISHED' in normalized_a_values:
+            refereed = False
+
+    self['refereed'] = refereed
+    return proceedings
+
+
+@journals.over('short_title', '^711..')
+def short_title(self, key, value):
+    return value.get('a')
+
+
+@journals.over('title_variants', '^730..')
+@utils.for_each_value
+def title_variants(self, key, value):
```

Comment:
the value should be skipped if it has a `b`.

Reply:
This now works, and there's a new test to assert this behavior.